### PR TITLE
Set up the landing page

### PR DESCRIPTION
**Pull request by Adnanog**

### CODE REVIEW

File: client/src/components/Home.js [hunk 1]
```diff
@@ -1,13 +1,43 @@
 import React from "react";
 import "../styles/Home.css";
 import { Header } from "semantic-ui-react";
```
Comment (islam95) [inline client/src/components/Home.js]: You can put these two lines in one

File: client/src/components/Home.js [hunk 2]
```diff
@@ -1,13 +1,43 @@
...
+import { Menu } from "semantic-ui-react";
 
 const Home = () => {
   return (
     <div>
       <header className="home-header">
```
Comment (islam95) [inline client/src/components/Home.js]: You imported Menu from Semantic UI but I don't see it you used in this file.
Also, there is Header that is not used. You can to use Semantic UI for Header, and any other html tags in your file.
You can have a look how it should be used here https://react.semantic-ui.com/elements/header/

File: client/src/styles/Home.css [hunk 3]
```diff
@@ -1,10 +1,67 @@
...
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
+  font-size: calc(10px + 0.5vmin);
   color: white;
 }
+.logo {
+  float: left;
```
Comment (islam95) [inline client/src/styles/Home.css]: If you use Semantic UI most of the CSS here will not be needed.

**Review by islam95 — CHANGES_REQUESTED** [review]

File: client/src/components/Home.js (hunk 1 above)
Comment (Adnanog) [reply client/src/components/Home.js]: Hi Islam, thanks for your comments, could you please tell me which two lines?are they  2, and 3 ?

File: client/src/components/Home.js (hunk 2 above)
Comment (Adnanog) [reply client/src/components/Home.js]: yes , I have deleted the Menu from the code.
the Header now has been used in the header tag and logo,but the font color of the header changed to black in opposite to the required wire-frame:( so how can I return it to white?

File: client/src/styles/Home.css (hunk 3 above)
Comment (Adnanog) [reply client/src/styles/Home.css]: I have a problem in setting the main image in the landing page , I tried to stretch it to be as the width of the page but it did not respond also I want to put it in the center of the page and also no response :(

File: client/src/components/Home.js (hunk 1 above)
Comment (islam95) [reply client/src/components/Home.js]: 2 lines that are similar to each other :)

File: client/src/components/Home.js [hunk 4]
```diff
@@ -5,9 +5,36 @@ import { Header } from "semantic-ui-react";
...
+        <p className="land-p-two">
+          They can also upload invites and other texts for translation and
+          distribution. As a stretch goal, Users can also use the app as a
+          platform to upload texts or photos of texts in another language that
+          they do not understand, including what we commonly nickname
+          “legalese”.
+        </p>
+        <button className="signUp">Sign Up</button>
+        <button className="interpreter">Become an Interpreter</button>
+      </div>
```
Comment (llh1) [inline client/src/components/Home.js:40]: Can you use primary button from Semantic UI here? So we have consistent styling. Check this here: https://react.semantic-ui.com/elements/button/#types-emphasis. You can remove the signUp and interpreter styles in the css after.

**Review by llh1 — CHANGES_REQUESTED** [review]

File: client/src/components/Home.js (hunk 4 above)
Comment (Adnanog) [reply client/src/components/Home.js:40]: I've used it already but the issue is that the locations are in undesired places and could not be modified by css as it is overridden by semantic.

Comment (llh1) [reply client/src/components/Home.js:40]: You can pass a className to the Semantic UI buttons, which center the buttons. Alternatively, you could also wrap these buttons into a div that you center on the screen.

**Review by llh1 — APPROVED** [review]

**Pull request merged**